thorlcr/slave/thslavemain.cpp

```diff
@@ -362,11 +373,10 @@ int main( int argc, char *argv[]  )
         else 
             slfEp.setLocalHost(0);
 
-        mySlaveNum = globals->getPropInt("@SLAVENUM");
-
+        // TBD: use new config/init system for generic handling of init settings vs command line overrides
+        if (0 == slfEp.port) // assume default from config if not on command line
+            slfEp.port = globals->getPropInt("@slaveport");
         setMachinePortBase(slfEp.port);
-        slfEp.port = getMachinePortBase();
```

Comment:
yes, it's used as the base port for dynamically allocating ports (e.g. SORT).

Reply:
Is it allowed to be 0 at this point?  Previously if it was zero it would have initialised it to THOR_BASESLAVE_PORT.  Now I think it will remain as 0.  Should the get() on line 378 have THOR_BASESLAVE_PORT as a default?

Reply:
probably doesn't make any practical difference, since slfEp is mainly used for tracing after this point,
but yes for consistency, if @slaveport is not defined, would be better to default it to THOR_BASESLAVE_PORT.